retrieval_service/datastore/__init__.py

```diff
@@ -21,7 +21,7 @@
     providers.firestore.Config,
     providers.postgres.Config,
     providers.cloudsql_postgres.Config,
-    providers.spanner_gsql.Config,
+    providers.spanner.Config,
```

Comment:
```suggestion
    providers.spanner.Config,
    providers.spanner_gsql.Config,
```

we are keeping both gsql and pg right?